src/runtime/lpython_builtin.py

```diff
@@ -775,6 +775,45 @@ def _lpython_str_startswith(s: str ,sub: str) -> bool:
         res = res and (j == len(sub))
     return res
 
+@overload
+def _lpython_str_endswith(s: str, suffix: str) -> bool: 
```

Comment:
These changes are already in the `main`, right?

If there is some problem,
- `Rebase` this branch on top of the updated main branch.
- Then do `git reset main` and commit the changes.